typescript/test/testdata/dlp/src/v2/dlp_service_client.ts.baseline

```diff
@@ -0,0 +1,3335 @@
+          protosTypes.google.privacy.dlp.v2.IUpdateJobTriggerRequest|undefined, {}|undefined>,
+      callback?: Callback<
+          protosTypes.google.privacy.dlp.v2.IJobTrigger,
+          protosTypes.google.privacy.dlp.v2.IUpdateJobTriggerRequest|undefined,
+          {}|undefined>):
+      Promise<[
+        protosTypes.google.privacy.dlp.v2.IJobTrigger,
+        protosTypes.google.privacy.dlp.v2.IUpdateJobTriggerRequest|undefined, {}|undefined
+      ]>|void {
+    request = request || {};
+    let options: gax.CallOptions;
+    if (typeof optionsOrCallback === 'function' && callback === undefined) {
+      callback = optionsOrCallback;
+      options = {};
+    }
+    else {
+      options = optionsOrCallback as gax.CallOptions;
+    }
+    options = options || {};
+    options.otherArgs = options.otherArgs || {};
+    options.otherArgs.headers = options.otherArgs.headers || {};
+    options.otherArgs.headers[
+      'x-goog-request-params'
+    ] = gax.routingHeader.fromParams({
+      'name': request.name || '',
+    });
+    return this._innerApiCalls.updateJobTrigger(request, options, callback);
+  }
+  getJobTrigger(
+      request: protosTypes.google.privacy.dlp.v2.IGetJobTriggerRequest,
+      options?: gax.CallOptions):
+      Promise<[
+        protosTypes.google.privacy.dlp.v2.IJobTrigger,
+        protosTypes.google.privacy.dlp.v2.IGetJobTriggerRequest|undefined, {}|undefined
+      ]>;
+  getJobTrigger(
+      request: protosTypes.google.privacy.dlp.v2.IGetJobTriggerRequest,
+      options: gax.CallOptions,
+      callback: Callback<
+          protosTypes.google.privacy.dlp.v2.IJobTrigger,
+          protosTypes.google.privacy.dlp.v2.IGetJobTriggerRequest|undefined,
+          {}|undefined>): void;
+/**
+ * Gets a job trigger.
+ * See https://cloud.google.com/dlp/docs/creating-job-triggers to learn more.
+ *
+ * @param {Object} request
+ *   The request object that will be sent.
+ * @param {string} request.name
+ *   Required. Resource name of the project and the triggeredJob, for example
+ *   `projects/dlp-test-project/jobTriggers/53234423`.
+ * @param {object} [options]
+ *   Call options. See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions} for more details.
+ * @returns {Promise} - The promise which resolves to an array.
+ *   The first element of the array is an object representing [JobTrigger]{@link google.privacy.dlp.v2.JobTrigger}.
+ *   The promise has a method named "cancel" which cancels the ongoing API call.
+ */
+  getJobTrigger(
+      request: protosTypes.google.privacy.dlp.v2.IGetJobTriggerRequest,
+      optionsOrCallback?: gax.CallOptions|Callback<
+          protosTypes.google.privacy.dlp.v2.IJobTrigger,
+          protosTypes.google.privacy.dlp.v2.IGetJobTriggerRequest|undefined, {}|undefined>,
+      callback?: Callback<
+          protosTypes.google.privacy.dlp.v2.IJobTrigger,
+          protosTypes.go
```

Comment:
casing is wrong here (should be `organizationInspectTemplate`, not `organizationinspecttemplate`)